src/python/WMCore/WMSpec/Steps/Executors/LogCollect.py

```diff
@@ -63,18 +63,18 @@ def execute(self, emulator = None):
 
         # Set wait to over an hour
         waitTime  = overrides.get('waitTime', 3600 + (self.step.retryDelay * self.step.retryCount))
-        seName    = overrides.get('seName',    "srm-cms.cern.ch")
-        pnn       = overrides.get('pnn',    "T2_CERN_CH")
-        lfnPrefix = overrides.get('lfnPrefix', "srm://srm-cms.cern.ch:8443/srm/managerv2?SFN=/castor/cern.ch/cms")
-        lfnBase   = overrides.get('lfnBase',   "/store/user/jsmith")
+        seName    = overrides.get('se-name', "srm-cms.cern.ch")
+        pnn       = overrides.get('phedex-node', "T2_CH_CERN")
```

Comment:
I didn't go that deep, but from what I saw, we set the default values because we (in Central Production, at least) never set any override values. I checked 3 or 4 workflows and all of them had an empty override section.